Makefile

```diff
@@ -509,6 +509,8 @@ security-test:
 .PHONY: common-test
 common-test:
 	go test ${T} ./pkg/...
+    # Execute bash shell unit tests scripts
+    ./tests/scripts/scripts_test.sh
```

Comment:
Might just be my display but the indent looks off? I think make is pretty strict about the indent

Reply:
Double checked and looks good on my side and if there were a problem `make` would have complained about it.